jira/resilientsession.py

```diff
@@ -290,11 +291,14 @@ def __recoverable(
         Returns:
             bool: True if the request should be retried.
         """
-        is_recoverable = False  # Controls return value AND whether we delay or not, Not-recoverable by default
+        suggested_delay = (
+            -1
+        )  # Controls return value AND whether we delay or not, Not-recoverable by default
```

Comment:
right now this comment doesn't quite make sense

renaming this var to `recovery_delay` would make it much more readable

Reply:
what would you change the comment to? (it makes sense to me but I'm ok to add more meaning to it)

I kind of agree with `recovery_delay`, but then it won't eventually be the recovery delay because of the max limit.

Reply:
then you go with -1 right? anyhow, this was just a minor suggestion, don't worry about it